src/perception/src/cone_evaluator/distance_predict.cpp

```diff
@@ -0,0 +1,28 @@
+#include <cone_evaluator/distance_predict.hpp>
+#include <cmath>
+
+#define CONE_WIDTH 0.228
+#define CONE_HEIGHT 0.325
+
+
+DistancePredict::DistancePredict(double vertical_ang_res, double horizontal_ang_res) :
+            vertical_ang_res(vertical_ang_res), horizontal_ang_res(horizontal_ang_res) {}
+
+
+double DistancePredict::evaluateCluster(Cluster& cluster) const {
+    double coneDistance = cluster.getCentroid().norm();
+
+    if (coneDistance <= 0) return 0;
+
+    double vertical_component = CONE_HEIGHT / (2.0 * coneDistance *
+                        tan(vertical_ang_res * M_PI / 180.0 / 2));
+    double horizontal_component = CONE_WIDTH / (2.0 * coneDistance *
+                        tan(horizontal_ang_res * M_PI / 180.0 / 2));
+
+    double n_points_prediction = 0.5 * vertical_component * horizontal_component;
```

Comment:
This is the number of points you think you should be receiving? The only think that concerns me here is that I don't see any part of the equations above taking the shape of the cone into account